tests/utils_tests/image_tests/test_tile_images.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import division
+import math
+import numpy as np
+import unittest
+
+from chainer import testing
+
+from chainercv.utils import tile_images
+
+
+@testing.parameterize(*testing.product({
+    'fill': [128, (104, 117, 123), np.random.uniform(255, size=(3, 1, 1))],
+    'pad': [1, 2, 3]
```

Comment:
How about testing the case `pad = 0`? With this test, [this bug](https://github.com/chainer/chainercv/pull/422/files/111bdd0fe06e3233a1edf59d7d0ac849b5407d75#r139073765) can be found.